src/dt/perspectives/stereotype/bias_generation.py

```diff
@@ -231,13 +224,6 @@ def main(OPTS):
                                  """Latin Americans""", """Christians""", """Canadians""",
                                  """Europeans""", """Americans""", """Caucasians""", """Buddhists"""]
 
```

Comment:
Please use double quotes.

Reply:
fixed in [49e36a9](https://github.com/AI-secure/DecodingTrust/pull/8/commits/49e36a92fa6823f6980ade70858871139410e42e)